ERCS/erc-wallet-asset-discovery.md

```diff
@@ -0,0 +1,218 @@
+---
+eip: xxxx
```

Comment:
```suggestion
eip: 7811
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.